src/sprightly.ts

```diff
@@ -58,28 +83,42 @@ function sprightly(
     ...defaultOptions,
     ...options,
   }
-  const entryPointAbsolutePath = path.resolve(path.normalize(entryPoint.trim()))
-  if (options.cache && cache.has(entryPointAbsolutePath))
-    return cache.get(entryPointAbsolutePath) as string
+
+  console.log(path.resolve(entryPoint))
+  const entryPointAbsolutePath = path.resolve(path.normalize(entryPoint))
```

Comment:
I think you forgot to address this https://github.com/obadakhalili/sprightly/pull/4#discussion_r978353593.